### PR TITLE
Adds car-free day as a holiday

### PR DESCRIPTION
**Pull request by ghost**
:cl: Qustinnus
add: Adds car-free day as a holiday. It means you get a free car.
/:cl:

I was very ecstatic when I saw that there was a car-free day, because I had been saving up for a car for quite some time, but apparently they give them out for free once a year.

This adds this same holiday to tg. On September 22 everyone spawns with a beacon to get a free car. These cars are a slow to get into but quick transportation method for around the station, They don't kidnap people like clown cars but are still useful for quick travel through the station!

### CODE REVIEW

File: code/modules/holiday/holidays.dm [hunk 1]
```diff
@@ -312,11 +312,16 @@
...
+	name = CARFREEDAY
+	begin_day = 22
+	begin_month = SEPTEMBER
+
+/datum/holiday/carfreeday/greet()
+	return "Car-free day! Damn, and to think I was saving up for one."
 
 /datum/holiday/pirate/getStationPrefix()
-	return pick("Yarr","Scurvy","Yo-ho-ho")
+	return pick("Vroom","Jam","Crash")
```
Comment (Jared-Fogle) [inline code/modules/holiday/holidays.dm:324]: You're editing the pirate prefixes.

File: code/modules/vehicles/cars/civcar.dm [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+/obj/item/car_beacon/attack_self()
+	if(used)
+		return
+	loc.visible_message("<span class='warning'>\The [src] begins to beep loudly!</span>")
+	used = TRUE
+	addtimer(CALLBACK(src, .proc/launch_payload), 40)
+
+/obj/item/car_beacon/proc/launch_payload()
+	var/obj/structure/closet/supplypod/centcompod/toLaunch = new()
+
```
Comment (MrDoomBringer) [inline code/modules/vehicles/cars/civcar.dm:107]:
```suggestion
toLaunch.setStyle(STYLE_SEETHROUGH)
toLaunch.delays[POD_OPENING] = 0
```

Comment (MrDoomBringer) [reply code/modules/vehicles/cars/civcar.dm:107]: commit to the goofiness and make the car fall in from the sky

**Review by MrDoomBringer — CHANGES_REQUESTED** [review]

File: code/modules/holiday/holidays.dm [hunk 3]
```diff
@@ -312,11 +312,16 @@
 	begin_month = SEPTEMBER
 	drone_hat = /obj/item/clothing/head/pirate
 
-/datum/holiday/pirate/greet()
-	return "Ye be talkin' like a pirate today or else ye'r walkin' tha plank, matey!"
```
Comment (Jared-Fogle) [inline code/modules/holiday/holidays.dm:316]: You're DELETING THE PIRATE HOLIDAY FLOYD WHERES IT GOING

Comment (Jared-Fogle) [reply code/modules/holiday/holidays.dm:316]: YOURE STILL DELETING THE GREET

Comment (Jared-Fogle) [reply code/modules/holiday/holidays.dm:316]: WHAT DID PIRATE DAY DO TO YOU

Comment (Qustinnus) [reply code/modules/holiday/holidays.dm:316]: NOOOOOOOOOOOOOOOOOOOOO

File: code/modules/holiday/holidays.dm [hunk 4]
```diff
@@ -312,11 +312,22 @@
 	begin_month = SEPTEMBER
 	drone_hat = /obj/item/clothing/head/pirate
 
+/datum/holiday/pirate/getStationPrefix()
+	return pick("Yarr","Scurvy","Yo-ho-ho")
+
```
Comment (Cyprex) [inline code/modules/holiday/holidays.dm:317]: Undocumented change, very minor though

Comment (Qustinnus) [reply code/modules/holiday/holidays.dm:317]: I just moved it

**Comment by Ziiro** [comment]
Even though they can't kidnap, can they still run people down a la MULE?

**Comment by trexter555** [comment]
make it so when two cars collide they explode killing both people driving them

File: code/modules/holiday/holidays.dm [hunk 5]
```diff
@@ -312,11 +312,22 @@
...
+/datum/holiday/carfreeday
+	name = CARFREEDAY
+	begin_day = 22
+	begin_month = SEPTEMBER
+
+/datum/holiday/carfreeday/greet()
+	return "Car-free day! Damn, and to think I was saving up for one."
+
+/datum/holiday/carfreeday/getStationPrefix()
+	return pick("Vroom","Jam","Crash")
```
Comment (MrDoomBringer) [inline code/modules/holiday/holidays.dm:330]:
```suggestion
	return pick("Vroom","GT","Crash","Sport Utility","High Octane","High Speed Low Drag","3 Door Fujiwara Sprinter Trueno")
```
VERY important

**Review by MrDoomBringer — CHANGES_REQUESTED** [review]

File: code/modules/holiday/holidays.dm (hunk 5 above)
Comment (ATH1909) [reply code/modules/holiday/holidays.dm:330]: add honda civic to the list too

File: code/modules/vehicles/cars/civcar.dm [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+	. = ..()
+	if(!isliving(AM))
+		return
+	var/mob/living/L = AM
+	if(!(L.mobility_flags & MOBILITY_STAND) && !L.buckle_lying)
+		for(var/i in return_drivers())
+			if(!ishuman(i))
+				continue
+			var/mob/living/carbon/human/boomer = i
+			boomer.say(pick("DON'T LIE IN FRONT OF MY CAR YA FUCKIN' KNOB!!", "HEY! I'M DRIVIN' HERE!!", "YOU ARE RUINING MY SUSPENSION, CUNT!"), forced="car crash")
```
Comment (Iamgoofball) [inline code/modules/vehicles/cars/civcar.dm:36]: can't have that on our github

File: code/modules/vehicles/cars/civcar.dm [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+/obj/vehicle/sealed/car/civ/Bump(atom/A)
+	. = ..()
+	if(!isliving(A))
+		return
+	var/mob/living/L = A
+	for(var/i in return_drivers())
+		if(!ishuman(i))
+			continue
+		var/mob/living/carbon/human/boomer = i
+		boomer.say(pick("HOLY SHIT MY PAINT IS RUINED!!", "I LEASED THIS YOU DICK!!", "GET OUT OF THE WAY! I WILL END YOU!!", "WHAT THE FUCK!? DO YOU KNOW HOW EXPENSIVE CARS ARE?!!", "GET OFF THE ROAD YOU BRAINDEAD TROGLODYTE!!", "GET OUT OF THE GODDAMN WAY!!"), forced="car crash")
```
Comment (Iamgoofball) [inline code/modules/vehicles/cars/civcar.dm:49]: imho move these to a json

File: code/modules/vehicles/cars/civcar.dm [hunk 8]
```diff
@@ -0,0 +1,112 @@
...
+	. = ..()
+
+/obj/vehicle/sealed/car/civ/generate_actions()
+	. = ..()
+	initialize_controller_action_type(/datum/action/vehicle/sealed/horn, VEHICLE_CONTROL_DRIVE)
+
+/obj/vehicle/sealed/car/civ/Cross(atom/movable/AM)
+	. = ..()
+	if(!isliving(AM))
+		return
```
Comment (Iamgoofball) [inline code/modules/vehicles/cars/civcar.dm:29]: so i can't drive over borgs?

File: code/modules/vehicles/cars/civcar.dm [hunk 9]
```diff
@@ -0,0 +1,112 @@
+
+/obj/vehicle/sealed/car/civ
+	name = "car"
+	desc = "This used to be a pedestrian-focused station, and now all these cars are ruining it!" //ok boomer
+	icon_state = "car"
+	max_integrity = 300
+	armor = list("melee" = 70, "bullet" = 40, "laser" = 40, "energy" = 0, "bomb" = 30, "bio" = 0, "rad" = 0, "fire" = 80, "acid" = 80)
```
Comment (Iamgoofball) [inline code/modules/vehicles/cars/civcar.dm:7]: these are extremely tanky cars

File: code/modules/vehicles/cars/civcar.dm (hunk 8 above)
Comment (Ghommie) [reply code/modules/vehicles/cars/civcar.dm:29]: silicons/borgs are a living mob subtype.

File: code/modules/vehicles/cars/civcar.dm (hunk 9 above)
Comment (MrDoomBringer) [reply code/modules/vehicles/cars/civcar.dm:7]: goof have you ever tried engaging a car in melee combat

Comment (Iamgoofball) [reply code/modules/vehicles/cars/civcar.dm:7]: according to fallout 3, 4, and 76 this immediately results in a massive explosion

**Comment by Iamgoofball** [comment]
as an American, 👍 :shipit:

File: code/modules/vehicles/cars/civcar.dm (hunk 9 above)
Comment (Fikou) [inline code/modules/vehicles/cars/civcar.dm:7]: use the armor defines

File: code/modules/vehicles/cars/civcar.dm (hunk 6 above)
Comment (wesoda25) [reply code/modules/vehicles/cars/civcar.dm:36]: Why?

**Comment by optimumtact** [comment]
I don't want to normalize silly holiday gifts, so I will deny this

**Pull request closed without merging**